user_manual/groupware/calendar.rst

```diff
@@ -2,28 +2,47 @@
 Using the Calendar app
 ======================
 
-.. note:: The calendar app is not enabled by default and needs to be installed
-          separately from our App Store. Please ask your Administrator for it.
+.. note:: The Calendar app comes installed with Nextcloud Hub by default, but can be disabled.
```

Comment:
We should say the dav backend is always active even if the app is missing or disabled.

Reply:
This is the user manual, not the dev / admin one. I don't think users need this info tbh.

Reply:
We can remove the whole note then.

Reply:
I would vote for leaving it here. Many users have old NC version in use and read the docs.

Reply:
@rakekniven I don't see the connection

Reply:
I also don't think the info about the dav back-end is relevant for users. But the note about getting in contact with their admin if they don't have a calendar is still relevant. I would leave the note. It was only adapted because calendar is now installed by default. In the past is was not.